tests/App/Admin/AuthorAdmin.php

```diff
@@ -71,6 +80,16 @@ protected function configureFormFields(FormMapper $form): void
             ]);
     }
 
+    protected function configureExportFields(): array
+    {
+        return [
+            'id',
```

Comment:
Maybe you can add a test for virtual fields like `count(address.id) as addressCount`.

Reply:
Im trying to debug: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/1701

Hopefully I will find the bug and fix it, and this could be the foundation for future tests that ensure things related with exports works correctly.